### PR TITLE
feat(selectionAssistant): introduce SelectionAssistant

### PR DESCRIPTION
**Pull request by ivoplashkov**
- Introducing a utility function, `getElementSelection`, designed to capture and return the coordinates and selected text from Input and TextArea elements. 

- The utility is particularly useful for scenarios where custom styling or actions are needed based on user text selection, such as creating custom tooltips, annotations, or other interactive features.

### CODE REVIEW

File: packages/website/docs/components/patterns/SelectionAssistant/InputSelectionAssistant/Basic/sample.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+		#btn {
+			display: none;
+			position: absolute;
+		}
+	</style>
+	<title>Sample</title>
+</head>
+
+<body style="background-color: var(--sapBackgroundColor); height: 300px;">
+	<!-- playground-fold-end -->
```
Comment (elenastoyanovaa) [inline packages/website/docs/components/patterns/SelectionAssistant/InputSelectionAssistant/Basic/sample.html:18]: is this correct?

File: packages/website/docs/components/patterns/SelectionAssistant/Selection Assistant.mdx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import Input from "../../../components/patterns/SelectionAssistant/InputSelectionAssistant/Basic/Basic.md";
+import TextArea from "../../../components/patterns/SelectionAssistant/TextAreaSelectionAssistant/Basic/Basic.md";
+
+**Note:** The util is in an experimental state
+
+### ES6 Module Import
+
+`import getElementSelection from "@ui5/webcomponents-base/dist/util/SelectionAssistant.js";`
+
+The sample demonstrates the usage of <strong>SelectionAssistant</strong> that enhances user interactions by
```
Comment (elenastoyanovaa) [inline packages/website/docs/components/patterns/SelectionAssistant/Selection Assistant.mdx:14]: @LilyanaOviPe could you please check the documentation?

**Review by elenastoyanovaa — CHANGES_REQUESTED** [review]

**Review by elenastoyanovaa — CHANGES_REQUESTED** [review]
- FF: the button is not showing in the samples
- Safari: the samples are not loaded at all, seems to be a general problem of the playground, not those specific samples. Nevertheless, I have no idea if the util is working on this browser.

File: packages/website/docs/components/patterns/SelectionAssistant/Selection Assistant.mdx (hunk 2 above)
Comment (LilyanaOviPe) [inline packages/website/docs/components/patterns/SelectionAssistant/Selection Assistant.mdx:14]: The sample demonstrates the usage of <strong>SelectionAssistant</strong>, which enhances user interactions by

File: packages/website/docs/components/patterns/SelectionAssistant/TextAreaSelectionAssistant/Basic/sample.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+			display: none;
+			position: absolute;
+		}
+	</style>
+	<title>Sample</title>
+</head>
+
+<body style="background-color: var(--sapBackgroundColor); height: 300px;">
+	<!-- playground-fold-end -->
+	<ui5-label for="ai-textarea">Textarea with selection assistant.</ui5-label>
```
Comment (LilyanaOviPe) [inline packages/website/docs/components/patterns/SelectionAssistant/TextAreaSelectionAssistant/Basic/sample.html:19]: TextArea with Selection Assistant.

File: packages/website/docs/components/patterns/SelectionAssistant/InputSelectionAssistant/Basic/sample.html (hunk 1 above)
Comment (ivoplashkov) [reply packages/website/docs/components/patterns/SelectionAssistant/InputSelectionAssistant/Basic/sample.html:18]: Seems to be the same for all the other samples.

**Review by elenastoyanovaa — APPROVED** [review]

**Pull request merged**